target/linux/meson/files/arch/arm/boot/dts/meson8b-onecloud.dts

```diff
@@ -0,0 +1,389 @@
+	clocks = <&xtal>, <&xtal>;
+	clock-names = "clkin0", "clkin1";
+};
+
+&cpu0 {
+	cpu-supply = <&vcc_core>;
+};
+
+
+&gpio {
+	gpio-line-names =
+		// 0-3 Bank GPIOX 0-3
+		"7U1_PIN_18", "7U1_PIN_19",
+		"7U1_PIN_14", "7U1_PIN_15",
+		// 4-7 Bank GPIOX 4-7
+		"7U1_PIN_27", "7U1_PIN_25",
+		"7U1_PIN_28", "7U1_PIN_26",
+		// 8 Bank GPIOX 8
+		"R_RightOf_LED",
+		// 9 Bank GPIOX 9
+		"7U1_PIN_16",
+		// 10 Bank GPIOX 10
+		"",
+		// 11 Bank GPIOX 11
+		"7U1_PIN_12",
+		// 12-16 Bank GPIOX 16-20
+		"7U1_PIN_43", "7U1_PIN_42",
+		"7U1_PIN_41(Resistor)", "7U1_PIN_44",
+		"7U1_PIN_34",
+		// 17 Bank GPIOX 21
+		"7U1_PIN_13",
+
+		// 18 Bank GPIOY 0
+		"Resistor_TopOf_LED",
+		// 19-20 Bank GPIOY 1-3
+		"", "",
+		// 21-26 Bank GPIOY 6-11
+		"", "",
+		"", "",
+		"", "",
+		// 27 Bank GPIOY 12
+		"",
+		// 28-29 Bank GPIOY 13-14
+		"Left_BottomOf_CPU",  "Right_BottomOf_CPU",
+
+		// 30 Bank GPIODV 9
+		"VCCK_PWM (PWM_C)",
+		// 31-35 Bank GPIODV 24-28
+		"I2CA_SDA", "I2CA_SCL",
+		"I2CB_SDA", "I2CB_SCL",
+		"VDDEE_PWM (PWM_D)",
+		// 36 Bank GPIODV 29
+		"",
+
+		// 37-39 Bank GPIOH 0-2
+		"HDMI_HPD", "HDMI_I2C_SDA", "HDMI_I2C_SCL",
+		// 40-46 Bank GPIOH 3-9
+		"ETH_PHY_INTR", "ETH_PHY_NRST",
+		"ETH_TXD1", "ETH_TXD0",
+		"ETH_TXD3", "ETH_TXD2",
+		"ETH_RGMII_TX_CLK",
+
+		// 47-53 Bank CARD 0-6
+		"SD_D1", "SD_D0",
+		"SD_CLK", "SD_CMD",
+		"SD_D3", "SD_D2",
+		"SD_CD",
+
+		// 54-65 Bank BOOT 0-11
+		"EMMC_D0", "EMMC_D1",
+		"EMMC_D2", "EMMC_D3",
+		"EMMC_D4", "EMMC_D5",
+		"EMMC_D6", "EMMC_D7",
+		"EMMC_CLK", "EMMC_RSTn",
+		"EMMC_CMD", "BOOT_SEL",
+		// 66-72 Bank BOOT 12-18
+		"", "",
+		"", "",
+		"", "",
+		"",
+
+		// 73-74 Bank DIF 0P 0N
+		"ETH_RXD1", "ETH_RXD0",
+		// 75-76 Bank DIF 1P 1N
+		"ETH_RX_DV", "RGMII_RX_CLK",
+		// 77-78 Bank DIF 2P 2N
+		"ETH_RXD3", "ETH_RXD2",
+		// 79-80 Bank DIF 3P 3N
+		"ETH_TXEN", "ETH_PHY_REF_CLK_25MOUT",
+		// 81-82 Bank DIF 4P 4N
+		"ETH_MDC", "ETH_MDIO";
+};
+
+&gpio_ao {
+	gpio-line-names =
+		// 0-1 Band GPIOAO 0-1
+		"UART TX", "UART RX",
+		// 2-4 Band GPIOAO 2-4
+		"RED_LED", "GREEN_LED", "BLUE_LED",
+		// 5 Band GPIOAO 5
+		"BUTTON",
+		// 6 Band GPIOAO 6
+		"",
+		// 7 Band GPIOAO 7
+		"IR_IN",
+		// 8 Band GPIOAO 8-13
+		"", "",
+		"", "",
+		"", "",
+
+		// 14 Band GPIO_BSD_EN
+		"",
+		// 15 GPIO_TEST_N
+		"";
+};
+
+&ethmac {
+	status = "okay";
+
+	pinctrl-0 = <&eth_rgmii_pins>;
+	pinctrl-names = "default";
+
+	phy-handle = <&eth_phy>;
+	phy-mode = "rgmii-rxid";
+
+	mdio {
+		compatible = "snps,dwmac-mdio";
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		// Realtek RTL8211F (0x001cc916)
+		eth_phy: ethernet-phy@0 {
+			reg = <0>;
+
+			reset-assert-us = <10000>;
+			reset-deassert-us = <80000>;
+			reset-gpios = <&gpio GPIOH_4 GPIO_ACTIVE_LOW>;
+
+			interrupt-parent = <&gpio_intc>;
+			interrupts = <17 IRQ_TYPE_LEVEL_LOW>; // GPIOH 3
+		};
+	};
+};
+
+&saradc {
+	status = "okay";
+	vref-supply = <&vcc_1v8>;
+};
+
+// eMMC
+&sdhc {
+	status = "okay";
+
+	pinctrl-0 = <&sdxc_c_pins>;
```

Comment:
sort and drop blank lines, for example:
https://github.com/immortalwrt/immortalwrt/blob/46c4f39158c3a3a3b959008917665b265ca14a84/target/linux/rockchip/files/arch/arm64/boot/dts/rockchip/rk3588s-nanopi-r6s.dts#L365-L378

the same applies to sd_card_slot